modular/gater/request_context.go

```diff
@@ -248,10 +283,16 @@ func (r *RequestContext) verifyOffChainSignature(requestSignature string) (sdk.A
 		return nil, err
 	}
 
+	// check request integrity
```

Comment:
As discussed,  following check is no longer needed.

```
	if hex.EncodeToString(realMsgToSign) != signedMsg {
		log.CtxErrorw(r.ctx, "failed to check signed msg")
		return nil, ErrRequestConsistent
	}
```

Reply:
Done